lib/codeowners/cli/interactive_helpers.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Codeowners
+  module Cli
+    # Helpers for the CLI (ask) and (yes) methods
```

Comment:
I'd consider adding an extra message saying it exists to ignore the interactions if the `options[:interactive]` is true.